run.sh

```diff
@@ -1,20 +1,25 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-wget -q http://www.cdf.toronto.edu/~csc343h/winter/assignments/a2/imdb.ddl -O imdb.ddl || exit 2
-patch imdb.ddl < imdb.ddl.patch
```

Comment:
Because `prep.sh` is meant to be called once to download and patch `imdb.ddl`. Pointless if calling `run.sh` multiple times (which I was doing :smile: ).